.travis.yml

```diff
@@ -1,15 +1,27 @@
 language: go
 
-# Allow use in forks
-go_import_path: github.com/vektra/mockery
+os: 
+  - linux
+  - osx
+
+env:
+  - GO111MODULE=on CGO_ENABLED=0
 
 go:
-  - 1.11.x
-  - 1.12.x
```

Comment:
stretchr/testify still supports 1.11/1.12. Yes I know they're ancient but let's keep them in here for now.

https://github.com/stretchr/testify#supported-go-versions

Reply:
Surly this only effects the build of the mockery binary, rather than the supported Go output by the tool? We would only build mockery binary with the latest (1.14) so we could actually _remove_ all but 1 Go version to run the tests. 💁‍♂️ 

That said, I've added both 1.11 and 1.12 back in as requested.